iis/mymodule.cpp

```diff
@@ -818,11 +818,7 @@ CMyHttpModule::OnBeginRequest(
         goto Finished;
     }
 
-	// every 3 seconds we check for changes in config file
-	//
-	DWORD ctime = GetTickCount();
-
-	if(pConfig->m_Config == NULL || (ctime - pConfig->m_dwLastCheck) > 3000)
```

Comment:
does that mean right now on the run time, we update the config if config  changed

Reply:
yes